packages/antd/README.md

```diff
@@ -0,0 +1,83 @@
+[![npm][npm-shield]][npm-url]
+[![npm downloads][npm-dl-shield]][npm-dl-url]
+
+<br />
+<p align="center">
+  <h3 align="center">@react-jsonschema-form/antd</h3>
+
+  <p align="center">
+    Ant Design theme, fields and widgets for <a href="https://github.com/rjsf-team/react-jsonschema-form"><code>react-jsonschema-form</code></a>.
+    <br />
+    <a href="https://react-jsonschema-form.readthedocs.io/"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://rjsf-team.github.io/react-jsonschema-form/">View Playground (TBD)</a>
+    ·
+    <a href="https://github.com/rjsf-team/react-jsonschema-form/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/rjsf-team/react-jsonschema-form/issues">Request Feature</a>
+  </p>
+</p>
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+- [Roadmap](#roadmap)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Getting Started
+
+### Prerequisites
+
+- `antd == 3` (not tested yet with version 4)
```

Comment:
Do you envision having to make a separate theme to support antd 4, or can one theme support both?

Reply:
Unfortunately, antd 4 introduced breaking changes related to the Form component. I haven't looked into the details yet, but I think we need to fix a period of time where we support two themes, and then slowly deprecate antd 3.